requirements/edx/github.txt

```diff
@@ -46,15 +47,15 @@
 # Python libraries to install directly from github
 
 # Third-party:
-git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline==1.5.3
+git+https://github.com/jazzband/django-pipeline.git@d068a019169c9de5ee20ece041a6dea236422852#egg=django-pipeline==1.5.3
```

Comment:
I changed the owner on this because when you visit the original URL on GitHub, it takes you to jazzband.  The commits are the same.